admin_manual/configuration_files/encryption_configuration.rst

```diff
@@ -211,6 +216,26 @@ Key then you can change a user's password in the ownCloud Users panel to match
 their back-end password, and then, of course, notify the user and give them 
 their new password.
 
+.. _upgrading:
+
+Upgrading From ownCloud 8.0
+---------------------------
+
+The encryption backend has changed in ownCloud 8.1, so you must take some 
+additional steps to migrate encryption correctly. If you do not follow these 
+steps you may not be able to access your files.
+
+After your upgrade is complete, follow the steps in :ref:`enable_encryption` to 
+enable the new encryption system. Then click the **Start Migration** button 
```

Comment:
I would like to add some kind of "warning" here. The "Start Migration" button is there only for small installations where the admin doesn't have access to the console (This means really small hosting services which provides only a ftp upload or something similar). We really recommend to use the occ tool whenever the admin has access to a terminal, even if the "start migartion" button is available. Can you add this @carlaschroder ? Thanks!